components/eam/bld/namelist_files/use_cases/2000_cam5_av1c-04p2-MMF-2mom.xml

```diff
@@ -45,62 +38,39 @@
 <!-- Radiation bugfix -->
 <use_rad_dt_cosz>.true.</use_rad_dt_cosz>
 
-<!-- Tunable parameters for 72 layer model -->
-
-<ice_sed_ai>         500.0  </ice_sed_ai>
-<cldfrc_dp1>         0.045D0</cldfrc_dp1>
-<clubb_ice_deep>     16.e-6 </clubb_ice_deep>
-<clubb_ice_sh>       50.e-6 </clubb_ice_sh>
-<clubb_liq_deep>     8.e-6  </clubb_liq_deep>  
-<clubb_liq_sh>       10.e-6 </clubb_liq_sh>
-<clubb_C2rt>         1.75D0 </clubb_C2rt>
-<zmconv_c0_lnd>      0.007  </zmconv_c0_lnd>
-<zmconv_c0_ocn>      0.007  </zmconv_c0_ocn>
-<zmconv_dmpdz>      -0.7e-3</zmconv_dmpdz>
-<zmconv_ke>          1.5E-6 </zmconv_ke>
-<effgw_oro>          0.25    </effgw_oro>
-<seasalt_emis_scale> 0.85   </seasalt_emis_scale>
-<dust_emis_fact>     2.05D0 </dust_emis_fact>
-<clubb_gamma_coef>   0.32   </clubb_gamma_coef>
-<clubb_C8>           4.3    </clubb_C8>
-<cldfrc2m_rhmaxi>    1.05D0 </cldfrc2m_rhmaxi>
-<clubb_c_K10>        0.3    </clubb_c_K10>
-<clubb_c_K10h>        0.3    </clubb_c_K10h>
-<effgw_beres>        0.4    </effgw_beres>
-<do_tms>             .false.</do_tms>
-<so4_sz_thresh_icenuc>0.075e-6</so4_sz_thresh_icenuc>
-<n_so4_monolayers_pcage>8.0D0 </n_so4_monolayers_pcage>
-<micro_mg_accre_enhan_fac>1.5D0</micro_mg_accre_enhan_fac>
-<zmconv_tiedke_add       >0.8D0</zmconv_tiedke_add>
-<zmconv_cape_cin         >1</zmconv_cape_cin>
-<zmconv_mx_bot_lyr_adj   >2</zmconv_mx_bot_lyr_adj>
+<!-- Tunable parameters -->
+<effgw_oro               >0.25    </effgw_oro>
+<seasalt_emis_scale      >0.85   </seasalt_emis_scale>
+<dust_emis_fact          >2.05D0 </dust_emis_fact>
+<effgw_beres             >0.4    </effgw_beres>
+<so4_sz_thresh_icenuc    >0.075e-6</so4_sz_thresh_icenuc>
+<n_so4_monolayers_pcage  >8.0D0 </n_so4_monolayers_pcage>
 <taubgnd                 >2.5D-3 </taubgnd>
-<clubb_C1                >1.335</clubb_C1>
-<raytau0                 >5.0D0</raytau0>
-<prc_coef1               >30500.0D0</prc_coef1>
-<prc_exp                 >3.19D0</prc_exp>
-<prc_exp1                >-1.2D0</prc_exp1>
-<se_ftype                >2</se_ftype>
-<clubb_C14               >1.3D0</clubb_C14>
-<relvar_fix              >.true.</relvar_fix>
-<mg_prc_coeff_fix        >.true.</mg_prc_coeff_fix>
-<rrtmg_temp_fix          >.true.</rrtmg_temp_fix>
+<raytau0                 >5.0D0 </raytau0>
+
+<!-- <ice_sed_ai>         500.0  </ice_sed_ai> -->
```

Comment:
Any reason to keep these commented out values?

Reply:
I don't think so... I'll delete these.